docusaurus/docs/welcome/instructions/update-app-icon.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
What is still missing:

- Briefly describe the purpose of an app icon.
- Explain that automatic generation is not a silver bullet. Sometimes, designers have to perform manual adjustments to specific icons for them to be displayed according to the specification.
- Why do we recommend Android Asset Studio by romannurik in favor of the official [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio) for automatic generation? Explain.
- What should be the size of the app icon to provide to the automatic generation tools?
- List Android app icon sizes for each screen density. If there is an official resource which already has such list, reference it instead.
- Is there anything else we can use for iOS which may assist developers and/or designers in setting up an app icon? We shouldn't rely only on App Icon Generator which can be taken down in the future. Are there any other designer or developer tools we can use, e.g., CLIs for automatic generation, plugins or templates for design tools, etc.?
- Update ["Project Setup"](https://react-native-with.moxy.tech/docs/welcome/instructions/project-setup) page by adding "Create an app icon" step.

Reply:
> What is still missing:
> 
>     * ~~Briefly describe the purpose of an app icon. I think we already have a definition in the task's description in JIRA.~~
> 
>     * Explain that automatic generation is not a silver bullet. Sometimes, designers have to perform manual adjustments to specific icons for them to be displayed according to the specification. Nonetheless, automatic generation should always be the first step.
> 
>     * Why do we recommend Android Asset Studio by romannurik in favor of the official [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio) for automatic generation? Explain.
> 
>     * What should be the size of the app icon to provide to the automatic generation tools?
> 
>     * List Android app icon sizes for each screen density. If there is an official resource which already has such list, reference it instead.
> 
>     * Is there anything else we can use for iOS which may assist developers and/or designers in setting up an app icon? We shouldn't rely only on App Icon Generator which can be taken down in the future. Are there any other designer or developer tools we can use, e.g., CLIs for automatic generation, plugins or templates for design tools, etc.?
> 
>     * ~~Update ["Project Setup"](https://react-native-with.moxy.tech/docs/welcome/instructions/project-setup) page by adding "Create an app icon" step.~~

Reply:
@guilhermedeandrade we should probably also have a description on how to generate the app icons for Android using the official Image Asset Studio precisely for the same reason I've stated on the iOS side of things.